java/README.md

```diff
@@ -0,0 +1,45 @@
+# Java samples and code
+
+To get this sample to run, you must [install OpenCV](http://opencv.org/downloads.html) on your local machine. You will need [Apache Ant](http://ant.apache.org) to build the executable JAR file app.  
+
+Your environment should define the following environment variables:
+
+* JAVA_HOME = the directory containing your JDK
+* ANT_HOME = the directory in which Apache Ant is installed
+* OPENCV_HOME = the directory in which all of OpenCV is installed
+* OPENCV_LIB = the directory containing all native libraries
+* OPENCV_JAR = the path to the JAR file containing the java interface to OpenCV (typically named something like "opencv-320.jar" )
+
+Execute the code by changing to the directory for GRIP-code-generation/java and then executing "ant run".   Or, can can run the application by executing:
+
+`java -Djava.library.path=${OPENCV_LIB} -jar build/grip_sample1.jar`
+
+## Installing OpenCV on Windows
```

Comment:
Could mention the opencv-installer program we've written that downloads and installs WPI's prebuilt bindings. No need to compile OpenCV yourself (for the most part -- it only does x86_64 windows, linux, and OS X, with support for generic linux armhf and rasbian coming soon)